package.json

```diff
@@ -287,7 +287,7 @@
 		"submodules": "cd resources && git submodule init && git submodule update && cd ..",
 		"pretest": "npm run submodules && npm run webpack && npm run lint",
 		"webpack": "webpack --mode development",
-		"webpack --watch": "webpack --mode development --watch"
+		"webpack-watch": "webpack --mode development --watch"

```

Comment:
for some reason the first naming won't run it in watch mode on mac :)

Reply:
ok np :)